includes/html/pages/device/apps/backupninja.inc.php

```diff
@@ -0,0 +1,22 @@
+<?php
+$graphs = array(
+    'backupninja_backupninja'       => 'Sauvegardes backupninja',
```

Comment:
Could you please write the text in english here (until this part of the LibreNMS gets updated to Laravel which takes care of the translations ?)